### DISCUSSION TITLE
Upgrade from 2020.5 to 2024.x receive "Access Denied"

### DISCUSSION THREAD

**Discussion by fryjkt1** [datalust/seq-tickets#2277] [Help]
We have upgraded our SEQ installation from 5.1 to 2020.5, as per the documentation.  We are attempting to upgrade to 2024.x, but are receiving "Access Denied" when running the .msi from an Administrator command prompt. Double clicking the .msi, nothing happens. 
 We tried to upgrade to several versions, including 2021.1 and various other versions, but receive the same error message.  Can you help troubleshoot?  Thank you!

**Comment by nblumhardt** [comment]
Hi! I'd suggest first re-starting the Windows host machine, if you haven't done that already. I'd also recommend checking that the Windows version is recent (at least Server 2016) and all updates are applied.

The next thing to try might then be to run the installer directly from an administrative PowerShell prompt (must be administrative) using `msiexec`:

```
msiexec /i Seq-nnnn.nn.nn.msi
```

`msiexec` has some diagnostics-related options that might help if that command fails - run `msiexec /?` to see more information.

Let me know if this helps,
Nick

**Comment by fryjkt1** [comment]


**Reply by nblumhardt** [reply]
> Nick,
> 
> Running an admin PowerShell window and executing the command that you suggested was the ticket.  We were able to successfully upgrade to 2024.x.  Than you very much for your assistance!

That's great, thanks for letting me know.

In case you were caught out by GitHub's misleading "Reply directly by email" wording - email replies still go to the public ticket. I edited your post to remove your email signature etc. but unfortunately Discussions preserves the edit history, so I'll delete it now instead.
